run/setup_codex_dev_env.sh

```
@@ -43,22 +43,22 @@
-# start services if not already running
-command -v chromadb >/dev/null || { echo "chromadb CLI not found on PATH"; exit 1; }
+# ensure PATH has the right scripts dir (see step 2)
+SCRIPTS_DIR="$(python -c 'import sysconfig; print(sysconfig.get_path("scripts"))')"
+case ":$PATH:" in *":$SCRIPTS_DIR:"*) : ;; *) export PATH="$SCRIPTS_DIR:$PATH" ;; esac
 
-pgrep -f 'chromadb run --host 127.0.0.1 --port 8000' >/dev/null || nohup chromadb run --host 127.0.0.1 --port 8000 >/dev/null 2>&1 &
+# start the server if not running
+
+
+pgrep -f 'uvx --from chromadb chroma run --host 127.0.0.1 --port 8000'  \\
+    ||nohup uvx --from chromadb chroma run --host 127.0.0.1 --port 8000 >/dev/null 2>&1 &
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Mismatched keywords or invalid parentheses?

(SC1070)